### ISSUE TITLE
CodeCrunch

### ISSUE THREAD

**Issue by jeronliaw** [nus-cs2030/2223-s1#321]
# Summary
Put a summary of the problem here

## Description 
Describe the problem and how one can replicate this problem 

## Screenshots (if any): 
Insert Images here if necessary

**Comment by Wenbo-J** [comment]
Agree, but for now you can try drawing out a dependency map, this helped me quite a lot!

**Comment by liewyungjun** [comment]
i actually found having the relationships shown quite useful
after that you can just ctrl f in your implementation to find where your cyclic dependencies are, cos sometimes when you code you accidentally put in the cyclic dependencies
